src/fah/webclient/index.html

```diff
@@ -3,481 +3,465 @@
+                        <li class="label-1 one-third column">Light</li>
+                        <li class="label-2 one-third column">Medium</li>
+                        <li class="label-3 one-third column">Full</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <div class="one-half column">
+                    <div id="box-idle-selector">
+                      <span>When</span>
+                      <ul class="radio-selector">
+                        <li>
+                          <input id="not_idle" name="idle" type="radio" value="1"
+                              checked="checked">
+                          <label for="not_idle">
+                            <span class="marker"></span><span>While I'm working</span>
+                          </label>
+                        </li>
+                        <li>
+                          <input id="idle" name="idle" type="radio" value="2">
+                          <label for="idle">
+                            <span class="marker"></span><span>Only when idle</span>
+                          </label>
+                        </li>
+                      </ul>
+                    </div>
+                  </div>
                 </div>
               </div>
+
+              <a id="btn-stop" href="#" class="button">Stop Folding</a>
+              <a id="btn-run" href="#" class="button" style="display:none">Start
+                Folding</a>
+              <p id="note-background">Quitting the browser will not stop folding.
+                Folding happens in the background on your computer through the program FAHClient
+                even when Web Control is closed.
+              </p>
             </div>
           </div>
+        </div>
+        <div class="one-half column">
+          <div id="box-stats-container">
+            <ul id="box-stats" class="row">
+              <li class="one-third column" id="box-stats-wu-col">
+                <div class="hlarge" id="box-stats-wu"></div>
+                <div>Work Unit (PRCG)</div>
+              </li>
+              <li class="one-third column" id="box-stats-days-col">
+                <div class="hlarge" id="box-stats-days"></div>
+                <div>Work Unit (ETA)</div>
+              </li>
+              <li class="one-third column" id="box-stats-points-col">
+                <div class="hlarge" id="box-stats-points"></div>
+                <div>Estimated Points</div>
+              </li>
+            </div>
 
-          <a id="btn-stop" href="#" class="button">Stop Folding</a>
-          <a id="btn-run" href="#" class="button" style="display:none">Start
-            Folding</a>
-          <p>Quitting the browser will not stop folding. Folding happens in a
-            background process on your computer through the program FAHClient
-            even when Web Control is closed.
-          </p>
+            <div id="box-project">Loading...</div>
+          </dl>
```

Comment:
Seems you have missed opening `<dl>` tag

Reply:
I think it is supposed to close the `ul` tag line 159
```suggestion
          </ul>
```